.github/workflows/pull-request-ci.yaml

```
@@ -0,0 +1,361 @@
+        DEBUG_MODE: false
+        GKE_NODE_COUNT: 1
+        GKE_PREEMPTIBLE: true
+        GKE_INSTANCE_TYPE: n1-highcpu-16
+
+    # Providing kubeconfig for debugging
+    - name: export KUBECONFIG
+      run: |
+        set -o errexit -o nounset -o pipefail
+        cd "build${BACKEND}"
+        source .envrc
+        echo "::set-env name=KUBECONFIG::${KUBECONFIG}"
+        # Debugging aid; should be safe as the cluster will be torn down
+        # at the end of the run.
+        cat "${KUBECONFIG}" | gzip | base64 --wrap=0
+      working-directory: catapult
+
+    # TBD: Don't generate values.yaml
+    - name: Generate KubeCF Configuration
+      run: |
+        set -o errexit -o nounset -o pipefail
+
+        make -C ${{ github.workspace }}/catapult kubecf-gen-config
+
+        cd ${{ github.workspace }}/catapult/build${BACKEND}
+
+        gomplate --context .=scf-config-values.yaml <<"EOF" \
+          > ${{ github.workspace }}/kubecf/dev/kubecf/kubecf-values.yaml
+        {{- /* Disable brain minibroker tests */}}
+        {{- define "minibroker-tests" }}
+        properties:
+          brain-tests:
+            acceptance-tests-brain:
+              tests:
+                minibroker:
+                  {{- range slice "mariadb" "mongodb" "postgres" "redis" }}
+                  {{ . }}:
+                    enabled: false
+                  {{- end }}
+        {{- end }}
+        {{- $ = tmpl.Exec "minibroker-tests" | yaml | merge $ }}
+
+        {{- /* Disable registry overrides */}}
+        {{- $ = coll.Omit "kube" $ }}
+
+        {{- /* Set DNS annotations */}}
+        {{- define "svc_annotation" }}
+        services:
+          {{ index . 0 }}:
+            annotations:
+              "external-dns.alpha.kubernetes.io/hostname": {{ index . 1 }}
+        {{- end }}
+        {{- $d := .system_domain }}
+        {{- $ = printf "%s, *.%s" $d $d | slice "router" | tmpl.Exec "svc_annotation" | yaml | merge $ }}
+        {{- $ = printf "ssh.%s" $d | slice "ssh-proxy" | tmpl.Exec "svc_annotation" | yaml | merge $ }}
+        {{- $ = printf "tcp.%s, *.tcp.%s" $d $d | slice "tcp-router" | tmpl.Exec "svc_annotation" | yaml | merge $ }}
+
+        {{- $ | toYAML }}
+        EOF
+      env:
+        BACKEND: gke
+        AUTOSCALER: "true"
+
+    # Helm install cf-operator
+    - run: make cf-operator-apply
+      env:
+        CF_OPERATOR_URL: ${{ github.workspace }}/kubecf-bundle/cf-operator.tgz
+
+    # Wait for cf-operator pods to be ready
+    - run: make cf-operator-wait
+
+    # Helm install kubecf
+    - run: make kubecf-apply
+      env:
+        CHART: ${{ github.workspace }}/kubecf-bundle/kubecf_release.tgz
+        FEATURE_EIRINI: ${{ matrix.backend == 'eirini' }}
+        VALUES: dev/kubecf/kubecf-values.yaml
+
+    # Wait for kubecf pods to be ready
+    - run: make kubecf-wait
+
+    # Run smoke tests
+    - run: make smoke-tests
+
+    # Run brain tests
+    - run: make brain-tests
+
+    # Run sits tests
```

```suggestion
    # Run sync integration tests
```
"Run sync integration test suite tests" is… technically valid, but rather verbose.  I'd also be fine with _removing_ the comments for the one-line commands, as they don't actually add any value.